pkg/pex/pex.go

```diff
@@ -33,7 +34,8 @@ const (
 	baseProto             = "/casm/pex"
 	Proto     protocol.ID = baseProto + "/" + Version
 
-	mtu = 2048 // maximum transmission unit => max capnp message size
+	mtu     = 2048 // maximum transmission unit => max capnp message size
+	timeout = time.Second * 15
```

Comment:
Can we add a comment indicating what this timeout controls?  Something as simple as `// push-pull` is good enough.